app/views/regions/import.html.erb

```diff
@@ -0,0 +1,8 @@
+<% provide(:title, "Importation des régions et départements") %>
+
+<h3>Regions in db</h3>
+<ul>
+  <% Region.all.each do |region| %>
+    <li><%= region.name %> (<%= Department.where(region: region).map(&:name).join(",") %>)</li>
```

Comment:
```suggestion
    <li><%= region.name %> (<%= region.departments.map(&:name).join(",") %>)</li>
```

Vu que le titre c'est "Regions in db" je pense qu'on peut effectivement montrer ce qu'il y a dans la db.

Reply:
En fait, cette vue aurait dû être supprimée ;)